src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -20,12 +20,12 @@
     @Autowired
     private CartMapper cartMapper;
 
-    @RequestMapping(method = RequestMethod.HEAD)
+    @PostMapping
     public Cart createEmptyCart() {
         return new Cart();
     }
```

Comment:
Nie jestem pewien ale wydaje mi się, że do utworzenia nowego koszyka powinieneś użyć metody save, którą zaimplementowałeś w klasie CartService

Reply:
jeśli mamy to już zaimplementowane to powinniśmy tego tutaj użyć

Reply:
ten pliczek już usunięty, teraz widzę :)